resources/user-management.yaml

```diff
@@ -168,4 +209,4 @@ components:
       properties:
         userId:
           type: string
-          format: UUID
+          format: UUID
```

Comment:
Since you are making this change I would remove the UUID format since a carbonio account could be created not in a UUID format (e.g.: zmprov ca u@example.com password zimbraId hello-guys